tools/gbench/report.py

```diff
@@ -171,46 +173,43 @@ def calc_utest(timings_cpu, timings_time):
 
     return (min_rep_cnt >= UTEST_OPTIMAL_REPETITIONS), cpu_pvalue, time_pvalue
 
-def print_utest(partition, utest_alpha, first_col_width, use_color=True):
+def print_utest(bc_name, n_of_measurements,
+                utest, utest_alpha, first_col_width, use_color=True):
     def get_utest_color(pval):
         return BC_FAIL if pval >= utest_alpha else BC_OKGREEN
 
-    timings_time = extract_field(partition, 'real_time')
-    timings_cpu = extract_field(partition, 'cpu_time')
-    have_optimal_repetitions, cpu_pvalue, time_pvalue = calc_utest(timings_cpu, timings_time)
-
     # Check if we failed miserably with minimum required repetitions for utest
-    if not have_optimal_repetitions and cpu_pvalue is None and time_pvalue is None:
+    if not utest['stable'] and utest['cpu_pvalue'] is None and utest['time_pvalue'] is None:
         return []
 
-    dsc = "U Test, Repetitions: {} vs {}".format(
-        len(timings_cpu[0]), len(timings_cpu[1]))
+    dsc = "U Test, Repetitions: {}".format(
+        n_of_measurements)
```

Comment:
Not happy about this. Can it not remain two numbers in some form?

Reply:
Yes, understood and I agree. I will code it up to remain consistent with previous output.